docs/en/contributor/Contributor-Getting-Started.md

```diff
@@ -132,12 +132,7 @@ When you create a pull request to address the issue, you should add a link/point
 issue itself. In order to do that you have to add certain text in the pull request description.
 For example, if your issue number is `#123`, you should include one of the following in your
 pull request description.
```

Comment:
```suggestion
For example, if your issue number is `#123`, the PR description should include `Fix new contributor task Alluxio/alluxio#123`.
```

also facepalm to this now deleted list of options

Reply:
List of options makes people confused(at least me facepalm)